src/main/java/uk/ac/ebi/biosamples/cohortatlas/model/FacetValue.java

```diff
@@ -6,6 +6,6 @@
 @Data
 @AllArgsConstructor
 public class FacetValue {
-  private String label;
```

Comment:
good to have label or value of something meaningful  than _id.

Reply:
_id is default output label for some of the aggragate queries in mongo db. If it is really important I can create a new Bean and convert it.

Reply:
I think so to keep the API meaningful. Atleast copying that field to "label" field

Reply:
`
    {
        "category": "dataTypes",
        "displayName": "Data Types",
        "searchPath": "dataTypes",
        "count": 5,
        "values": [
            {
                "count": 2,
                "label": "phenotypicData"
            },
            {
                "count": 1,
                "label": "genomicData"
            },
            {
                "count": 1,
                "label": "environmentalData"
            },
            {
                "count": 1,
                "label": "biospecimens"
            }
        ]
    },`